SPECS/cni-plugins/cni-plugins.spec

```diff
@@ -39,10 +39,13 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_default_cni_plugins_dir}/*
 
 %changelog
-* Wed Oct 13 2023 Mateusz Gozdek <mgozdek@microsoft.com> - 1.3.0-1
+* Wed Oct 18 2023 Mateusz Gozdek <mgozdek@microsoft.com> - 1.3.0-1
 - Make plugin binaries correctly print version
 - Upgrade to version 1.3.0
 
+* Mon Oct 16 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 0.9.1-16
```

Comment:
cni-pugins is at 1.3.0-1 in 2.0.
I think we should update it to 1.3.0-2, and update the changelog here, so that the newer version gets built with golang like you're wanting it to